storybook/src/components/FileList/FileInputWithFileList.tsx

```diff
@@ -0,0 +1,33 @@
+import { FileInput, FileList } from '@amsterdam/design-system-react'
+import { useRef, useState } from 'react'
+
+export const FileInputWithFileList = () => {
+  const inputRef = useRef<HTMLInputElement>(null)
+  const [files, setFiles] = useState<FileList | null>(null)
+
+  const handleFilesChange = () => {
```

Comment:
I think function names like `changeFiles` and `removeFile` are more straightforward.